agasc/agasc.py

```diff
@@ -12,6 +12,8 @@
 
 DATA_ROOT = ska_path('data', 'agasc')
 
+tables_open_file = getattr(tables, 'open_file', None) or tables.openFile
```

Comment:
Are the tables method changes a Python 3 issue, a PyTables API issue, or both (need newer PyTables for Python 3)?